profile/8100.json

```diff
@@ -0,0 +1,12 @@
+{
+    "process": "kanto_service",
+    "systemability": [
+        {
+            "name": 8100,
```

Comment:
Same question for the ID here. Is there some registry where these are coordinated?

Reply:
the id must be defined in `foundation/systemabilitymgr/samgr/interfaces/innerkits/samgr_proxy/include/system_ability_definition.h` I will create a patch that adds this ID, and specify it in the README

Reply:
Thanks